### DISCUSSION TITLE
要統一用 GCP server 嗎？

### DISCUSSION THREAD

**Discussion by stevengogogo** [stevengogogo/DSA_EmailSearcher#10] [Q&A] [answered]
GCP server 可以用 linux 系統, 4CPU/16GB -> 120NTU/Month

我的 Cproject template 可以用 debugger, 但是建立在 linux 系統和 vscode 

如果大家不同平台, 可能要刪除 .vscode (#9)

**Comment by stevengogogo** [comment, marked as answer]
gdb, Valgrind 和 gprof 在 linux 系統比較好 install

**Reply by jcgs2503** [reply]
可以用GCP喔 
我剛好有GCP的credit還蠻多的 我可以開一個專案邀請你們 這樣就不用付錢了

**Reply by stevengogogo** [reply]
讚也! 我現在 DSA 作業是用 Ubuntu 20.4 LTS

**Reply by stevengogogo** [reply]
好呀! 那我們把的 Public key 寄給你. GCP 有個選項可以設定 SSH 連線

**Reply by jcgs2503** [reply]
我先邀請你好了 Compute Engine 再給你設置

**Reply by stevengogogo** [reply]
再跟我說你的邀請路徑: 我的 gmail:
```
stevengogogo4321@gmail.com
```

**Reply by jcgs2503** [reply]
應該有寄一封信給你了喔

**Reply by stevengogogo** [reply]
> 應該有寄一封信給你了喔
**Got it!**

**Comment by stevengogogo** [comment]
Thanks @jcgs2503 for providing GCP credits 💰

The following VM is created:

> 8 CPU/32GB
> 227 NTD/Month

 

![Screen Shot 2021-05-19 at 5 14 46 PM](https://user-images.githubusercontent.com/29009898/118787788-b7ffe100-b8c5-11eb-9813-daf1535337f8.png)


Remember to close it on 6/27

**Reply by stevengogogo** [reply]
如何設定: Let's get on board! (#12)
